src/config/middleware/middleware.ts

```diff
@@ -0,0 +1,16 @@
+import helmet from "helmet";
+import bodyParser from "body-parser";
+import compression from "compression";
+import cookieParser from "cookie-parser";
+import cors from "cors";
+import type { Application } from "express";
+
+export const Middleware = (app: Application) => {
```

Comment:
naming conventions for functions is camel case. So you first letter of name should be in lowercase.

Reply:
Still should to be fixed. And don't forget about router